id3v2metadata.go

```diff
@@ -119,6 +119,14 @@ func (m metadataID3v2) Lyrics() string {
 	return t.(*Comm).Text
 }
 
+func (m metadataID3v2) Comment() string {
+	t, ok := m.frames[frames.Name("comment", m.Format())]
```

Comment:
I think this will always be empty because "comment" isn't in the map: https://github.com/dhowden/tag/blob/51cdc7da0283a840e45849b660c3bdd693e5fadd/id3v2metadata.go#L34.